lib/web/mage/sticky.js

```diff
@@ -41,8 +44,34 @@ define([
 
             if (!isStatic && this.element.is(':visible')) {
                 offset = $(document).scrollTop() - this.parentOffset;
+
+                if (typeof this.options.spacingTop === 'function') {
+                    offset += this.options.spacingTop();
+                } else {
+                    offset += this.options.spacingTop;
+                }
+
                 offset = Math.max(0, Math.min(offset, this.maxOffset));
-                this.element.css('top', offset);
+
+                if (offset &&
+                    this.options.offsetTop &&
+                    !this.element.hasClass(this.options.stickyClass)) {
+
+                    var offsetTop = 0;
+                    if (typeof this.options.offsetTop === 'function') {
+                        offsetTop = this.options.offsetTop();
+                    } else {
+                        offsetTop = this.options.offsetTop;
+                    }
```

Comment:
This pattern happens twice, and there may be further configurable numbers, so it probably should be separated into a private function.
At the top of this file:
```js
function optionToNumber(self, option) {
  var value = self.options[option] || 0;
  if (typeof value === 'function') {
    value = self.options[option]();
  }
  var converted = Number(value);
  if (isNaN(converted)) {
    throw Error('sticky: Could not convert supplied option "' + option + '" to Number from "' + value + '"');
  }
  return converted;
}
```

And then here, you can just have:
```js
var offsetTop = optionToNumber(this, 'offsetTop');
if (offset < offsetTop) {
  offset = 0;
}
```

And above, you can just have:
```js
offset += optionToNumber(this, 'spacingTop');
```